biome.json

```diff
@@ -29,6 +29,12 @@
     "enabled": true
   },
   "linter": {
-    "enabled": false
+    "enabled": true,
+    "rules": {
+      "suspicious": {
+        "noConsoleLog": "off",
+        "noExplicitAny": "off"
```

Comment:
What is the reason to disable this rule?

Reply:
There are a bunch of `any` that I didn't want to touch, so I turned off the rule. How do you want to proceed?

Reply:
Oh I didn't know we had them :/
Let's keep the rule `off` for now, and I'll double check the code, to get rid of those `any`s